app/controllers/katello/remote_execution_controller.rb

```diff
@@ -40,6 +40,11 @@ def inputs
           { :errata => params[:name] }
         elsif feature_name == 'katello_service_restart'
           { :helper => params[:name] }
+        elsif feature_name =~ /module_stream/
+          fail HttpErrors::NotFound, _('module streams not found') if params[:module_spec].blank?
+          inputs = { :module_spec => params[:module_spec] }
```

Comment:
Could we look into passing in `:action` too? I think @iNecas originally did this for package actions, where there's a separate template for each thing - install, update, etc, but I feel like the number of templates is getting out of control.  I don't quite understand why Katello can't use a single job template for all the features

Reply:
I don't recall any significant reason to chose one over the other. Should be doable.